### PR TITLE
test slack integration

### PR DESCRIPTION
**Pull request by cgundy**

### CODE REVIEW

**Comment by cgundy** [comment]
@cgundy test

**Comment by cgundy** [comment]
@cgundy test again

File: .github/workflows-source/ci-main.yml [hunk 1]
```diff
@@ -368,7 +362,7 @@ jobs:
 
   cargo-clippy-linux:
     name: Cargo Clippy Linux
-    <<: *dind-large-setup
```
Comment (marko-k0) [inline .github/workflows-source/ci-main.yml:371]: I'd leave this on `dind-large-setup`. It takes quite some resources so it's safer to have it on `dind-large-setup`.

Comment (cgundy) [reply .github/workflows-source/ci-main.yml:371]: Please ignore this PR, I was testing something out before, then reused it to test the slack integration

Comment (marko-k0) [reply .github/workflows-source/ci-main.yml:371]: @cgundy test.

Comment (marko-k0) [reply .github/workflows-source/ci-main.yml:371]: @dfinity/idx test

**Pull request closed without merging**